docs/release-notes/15-3-0/README.md

```diff
@@ -3,21 +3,102 @@ title: OpenProject 15.3.0
+
+This major feature introduces multiple improvements, with several underlying enhancements that refine scheduling, management, and handling of meeting series. All individual changes related to Recurring Meetings are listed in the full changelog at the end of these release notes.
+
+The following screenshot shows an example Meetings overview page, highlighting some of the changes for version 15.3:
+
+![Example screenshot of new Meetings overview page with OpenProject 15.3, including Recurring meetings / Meeting series](openproject-15-3-recurring-meetings-highlighted.png)
+
+Please particularly note that filtering between upcoming and past meetings now happens on top of the page instead of the sidebar on the left. The sidebar now distinguishes between My meetings / Recurring meetings and All meetings, with seperate sections for Meeting series and Involvement.
+
+### Create child work packages directly from the Relations tab
+
+With OpenProject 15.3, you can now again create child work packages directly from the Relations tab, making it faster and more intuitive to build structured work breakdowns.
+
+A new option, **New child**, has been added to the **+ Relation** dropdown. If child work packages already exist, a **+ Child** button appears in the header of the child work packages section and allows you to quickly add another child to the active work package. Selecting **New child** opens a dialog where you can immediately enter essential details like type, subject, description, and other mandatory fields. Once created, the child work package is added to the list, and a confirmation message appears.
+
+Screenshot
+
+### Include total sums in timesheet PDF exports for better insights
+
+Tracking logged time is now more transparent with total sum calculations in timesheet PDF exports. Previously, users had to manually sum up hours across multiple users. With OpenProject 15.3, the PDF export now includes:
+
+- A summary table on a separate page when exporting timesheets for multiple users, showing total hours per user and a final sum.
+- A sum row under each user’s timesheet table, ensuring quick insights into individual and team-wide logged hours.
+
+This enhancement simplifies reporting, making it easier to review and analyze tracked time across teams.
+
+Screenshot
+
+### Include child work packages in cost reports for more accurate tracking
+
+Cost reports in OpenProject now include child work packages, ensuring more comprehensive cost tracking. Previously, users managing hierarchical work packages (such as epics, features, or sub-tasks) had to manually select each work package to ensure full cost visibility.
+
+With OpenProject 15.3, a new **Include children** checkbox allows users to automatically pull in all nested work packages when generating cost reports. This eliminates the need for cumbersome manual selection and ensures that all relevant costs are accounted for in reports.
```

Comment:
Just FYI, this is not a checkbox it is the operator of the filter. You can either use "Work package is" (this will be only the selected workpackage) or "Work package is (includes descendants)" which will include the children, grandchildern, etc.
 
![Bildschirmfoto 2025-02-10 um 11 50 15](https://github.com/user-attachments/assets/47faaf88-0fd0-4846-9db9-959323bcba8f)